apps/filebrowser/src/filebrowser/api.py

```diff
@@ -129,4 +131,20 @@ def rename(request):
     raise Exception(_('The destination path "%s" already exists.') % dest_path)
 
   request.fs.rename(src_path, dest_path)
-  return HttpResponse(status=200)
\ No newline at end of file
+  return HttpResponse(status=200)
+
+@error_handler
+def content_summary(request, path):
+  path = _normalize_path(path)
```

Comment:
@nidhibhatg Does it currently work without this "normalization"?

@Harshg999  @JohanAhlen what do you think here, should we start by assuming that the `_normalize_path` will still be needed for the new APIs or should we start without it and add it later on if we find that ut actually is needed? If we are building an API from scratch now and we know who/when/where these endpoints are called there shouldn't, I think at least, be a need for these semi magical better-safe then sorry normalizations we have seen in the past....

Reply:
_normalize_path does 2 things:
- Fix the decoding of FS scheme correctly which was a recent addition by you.
- Appends a trailing slash (/) in the FS scheme if its missing one -- the comment says that this behaviour is seen in Kubernetes form factor where the ingress removes one slash.

Reply:
To test it out, I agree with you that let's remove it for some API and then test out that API in a Kubernetes based env and decide accordingly.

We need to choose some other API because we can't test this one out completely on Kubernetes based env as internally there no HDFS on that form factor and IIRC this API will work only for HDFS.